src/System.Net.Security/src/System/Net/Security/SslAuthenticationOptions.cs

```diff
@@ -24,15 +24,15 @@ internal SslAuthenticationOptions(SslClientAuthenticationOptions sslClientAuthen
             RemoteCertRequired = true;
             RemoteCertificateValidationCallback = sslClientAuthenticationOptions.RemoteCertificateValidationCallback;
             TargetHost = sslClientAuthenticationOptions.TargetHost;
+            CertSelectionDelegate = localCallback;
```

Comment:
Please remove a comment in SecureChannel.cs saying that this code never gets called (see where we call the callback)

Reply:
Good catch.

Reply:
I've added the comment you suggested in the latest commit.